elements/rh-tag/demo/outline-with-icon.html

```diff
@@ -1,20 +1,10 @@
-<rh-tag color="red" variant="outline" icon="information">Red</rh-tag>
-<rh-tag color="orange" variant="outline" icon="information">Orange</rh-tag>
-<rh-tag color="green" variant="outline">
-  Green
-  <svg slot="icon" height="1em" width: "1em" fill="currentColor" viewBox="0 0 512 512" aria-hidden="true" role="img">
-    <path d="M256 8C119.043 8 8 119.083 8 256c0 136.997 111.043 248 248 248s248-111.003 248-248C504 119.083 392.957 8 256 8zm0 110c23.196 0 42 18.804 42 42s-18.804 42-42 42-42-18.804-42-42 18.804-42 42-42zm56 254c0 6.627-5.373 12-12 12h-88c-6.627 0-12-5.373-12-12v-24c0-6.627 5.373-12 12-12h12v-64h-12c-6.627 0-12-5.373-12-12v-24c0-6.627 5.373-12 12-12h64c6.627 0 12 5.373 12 12v100h12c6.627 0 12 5.373 12 12v24z"></path>
-  </svg>
-</rh-tag>
-<rh-tag color="cyan" variant="outline" icon="information">Cyan</rh-tag>
-<rh-tag color="blue" variant="outline" icon="information">Blue</rh-tag>
-<rh-tag color="purple" variant="outline" icon="information">
-  Purple
-  <svg slot="icon" height="1em" width: "1em" fill="currentColor" viewBox="0 0 512 512" aria-hidden="true" role="img">
-    <path d="M256 8C119.043 8 8 119.083 8 256c0 136.997 111.043 248 248 248s248-111.003 248-248C504 119.083 392.957 8 256 8zm0 110c23.196 0 42 18.804 42 42s-18.804 42-42 42-42-18.804-42-42 18.804-42 42-42zm56 254c0 6.627-5.373 12-12 12h-88c-6.627 0-12-5.373-12-12v-24c0-6.627 5.373-12 12-12h12v-64h-12c-6.627 0-12-5.373-12-12v-24c0-6.627 5.373-12 12-12h64c6.627 0 12 5.373 12 12v100h12c6.627 0 12 5.373 12 12v24z"></path>
```

Comment:
same as in tag, are we making the slot capability for icons less visible?

Reply:
[c9c3b22](https://github.com/RedHat-UX/red-hat-design-system/pull/1853/commits/c9c3b2201964b16b79e6e0131a10a5bda6a4e76d)